deployment/ansible/roles/model-my-watershed.geoprocessing/handlers/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Restart Akka HTTP
```

Comment:
I think it would be best to choose less technology specific naming. Perhaps something more service specific like `Restart geoprocessing service`.